workflows/rnaseq/rnaseq-variant-calling.wdl

```diff
@@ -0,0 +1,132 @@
+# SPDX-License-Identifier: MIT
+# Copyright St. Jude Children's Research Hospital
+version 1.1
+
+import "../../tools/picard.wdl"
+import "../../tools/gatk4.wdl" as gatk
+
+workflow rnaseq_variant_calling {
+    meta {
+        description: "Call variants from RNA-Seq data. Produces a VCF file of variants. Based on GATK RNA-Seq short variant calling best practices pipeline."
+        outputs: {
+            recalibrated_bam: "BAM that has undergone recalibration of base quality scores"
+            recalibrated_bam_index: "Index file for recalibrated BAM file"
+            merged_vcf: "VCF of raw RNA-Seq variant calls"
+            merged_vcf_index: "Index for merged VCF file"
+            variant_filtered_vcf: "VCF file after variant filters have been applied"
+            variant_filtered_vcf_index: "Index for filtered variant VCF file"
+        }
+    }
+
+    parameter_meta {
+        bam: "BAM file of aligned RNA-Seq reads"
+        bam_index: "Index file for BAM file"
+        fasta: "Reference FASTA file"
+        fasta_index: "Index file for reference FASTA file"
+        dict: "Sequence dictionary for reference FASTA file"
+        callingIntervalList: "Interval list of regions from which to call variants. Used for parallelization."
+        knownVcfs: "Array of known indels VCF files"
+        knownVcfIndexes: "Array of index files for known indels VCF files"
+        dbSNP_vcf: "dbSNP VCF file"
+        dbSNP_vcf_index: "Index file for dbSNP VCF file"
+        scatterCount: "Number of intervals to scatter over"
+        output_vcf_name: "Name of output VCF file"
```

Comment:
IMO this should WF should `prefix` and force all the suffixes/extensions. I think if I'm reading correctly, the user doesn't have a say in the output BAM name.

Reply:
Agreed. I've made that change. Now `prefix` will be used to name the recalibrated BAM and the filtered VCF.